go/genkit/reflection.go

```diff
@@ -302,6 +357,10 @@ func handleRunAction(g *Genkit) func(w http.ResponseWriter, r *http.Request) err
 			json.Unmarshal(body.Context, &contextMap)
 		}
 
+		if err = resolveModel(ctx, g, body.Input); err != nil {
```

Comment:
Shouldn't this be in `runAction` after it fails to resolve the action and right before it returns an error saying it doesn't exist?